### ISSUE TITLE
plot -Sq: inconsistent behavior when saving labels to file

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#4218]
**Description of the problem**

The **plot -Sq** documentation says:

> +t[file]
> Save line label x, y, text to file [Line_labels.txt]. Use +T to save x, y, angle, text instead.

Running the following command:
```
gmt math -T0/360/1 T 5 MUL COSD 8 MUL = t.txt
gmt plot -R0/360/-10/10 -JM6i -W2p -Sqn4:+l"wiggly line"+tlocation.txt t.txt -pdf map
```
the output `location.txt` file has the following lines. The 4 columns are x, y, angle and text, inconsistent with the documentation:
```
72	7.99999999991	1.28463137576	wiggly line
144	7.99999999991	1.28463137576	wiggly line
216	7.99999999991	1.28463137576	wiggly line
288	7.99999999991	1.28463137576	wiggly line
```

Trying **+T**:
```
gmt math -T0/360/1 T 5 MUL COSD 8 MUL = t.txt
gmt plot -R0/360/-10/10 -JM6i -W2p -Sqn4:+l"wiggly line"+Tlocation.txt t.txt -pdf map
```
It gives the following error:
```
plot [ERROR]: Option -S: Parsing failure
```

**Comment by seisman** [comment]
One more thing, is it possible to only save the labels to a file, but don't plot lines and labels.

**Comment by PaulWessel** [comment]
I suspect I change my mind about having two different output formats.  People can easily use -i etc to exclude the angle.  So that is probably why you get an error on the defunct +T.  I prefer to declare a documentation bug - will check.
As for not drawing at all - not sure.  psxy has a default output of PostScript.

**Comment by seisman** [comment]
Closed by #4219
